components/status_dropdown/status_dropdown.scss

```diff
@@ -40,6 +40,12 @@
     }
 }
 
+#status-menu-dnd-timed_menuitem {
+    .SubMenuItem.MenuItem {
+        width: 100%;
```

Comment:
Is this how we do it on the rest of the elements? (in this submenu or in the menu before?)

I wonder if we are using in other places flex properties, or something like that.
My point is just to be coherent among the different elements / files.

Reply:
After a second look, I think the problem is not so much about taking 100%, but applying two shades at the same time. Maybe the submenu item has already a background and we are applying an extra background we shouldn't?

Reply:
I can see a difference in the shades between the element selected in the previous menu, and the one selected in this menu. Not sure if desired. Maybe ask someone from UX?

Reply:
@larkox problem wasn't colouring, when we were clicking on the item nothing was happening. What you are suggesting also make sense.